tap_google_analytics/ga_client.py

```diff
@@ -307,11 +317,10 @@ def process_response(self, response):
                         elif metric_type == 'number':
                             value = float(value)
 
-                        record[metric_name.replace("ga:","ga_")] = value
+                        record[metric_name.replace("ga:", "ga_")] = value
 
                 # Also add the [start_date,end_date) used for the report
-                record['report_start_date'] = self.start_date
-                record['report_end_date'] = self.end_date
+                record['report_date'] = report_date
```

Comment:
we should keep start_date and end_date as they are, since in the future if we want to allow the user to control chunk sizes (2 days, 3 days, etc..) we shouldn't need to do a schema change